batched-tx-disperse-multisend-py/src/constants.py

```diff
@@ -0,0 +1,5 @@
+"""Default values for the options / filters"""
```

Comment:
is this filtering out or filtering in tokens?

Reply:
I meant this to focus the scanning on a specific token: so a white list.
Could be otherwise ofc, even a list of addresses.
I thought about scanning for transfers of scammy / new tokens.

Reply:
i am always a bit weary of having static address lists. I would prefer to have the bot scan all. There is always the opportunity to filter out at a later point in time.